spikingjelly/activation_based/neuron/psn.py

```
@@ -252,42 +252,54 @@ def masked_weight(self):
             )
 
         weight = self.masked_weight()[
-            self.time_step,
-            self.time_step + 1 - self.queue.__len__() : self.time_step + 1,
+            time_step,
+            time_step + 1 - len(queue) : time_step + 1,
         ]
-        x_seq = torch.stack(self.queue)
+        x_seq = torch.stack(queue)
 
         for i in range(x.dim()):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)